README.md

```diff
@@ -1,14 +1,85 @@
 # tap-fitter
 
-RH Tap Fitter is a utility designed to aid operator developers to enable features and generate configurations in order to properly run in certain pipeline environments. 
+Tap Fitter is a utility designed to aid operator developers to generate the configuration files needed to properly run in certain pipeline environments.
 
-This CLI tool will injest composite templates and generate the appropriate devfiles.
+This CLI tool will ingest composite templates and generate the appropriate devfiles.
 
-To build & install:
-`make install`
+## Getting Started
 
-To test:
-`make test`
+To build the utility, run `make build` from the root directory of the tap-fitter project.
 
-Future To-Do:
-- Enable Tap Fitter as GitHub Action
+## Usage
+
+The `tap-fitter` command will ingest composite templates and generate corresponding devfiles.
+Note that because it uses relative paths to infer the destination directory to write the devfiles to, `tap-fitter` must be run from the destination directory that contains the catalog tree.
```

Comment:
Technically this is a limitation of the specific composite template used, not the tool.

Reply:
Maybe better if something like
```
```suggestion
Note: because templates may use relative paths to related artifacts, `tap-fitter` should be run in the same location as input templates.
```